test/e2e/framework/ginkgowrapper.go

```diff
@@ -246,6 +246,84 @@ func registerInSuite(ginkgoCall func(string, ...interface{}) bool, args []interf
 	return ginkgoCall(text, ginkgoArgs...)
 }
 
+var (
+	tagRe                 = regexp.MustCompile(`\[.*?\]`)
```

Comment:
why do you need a question mark in this refect? the `\[.*\]` is not enough?

Reply:
Without the question mark, two tags next to each other (`[foo][bar]`) get matched as one. The question mark ensures that matching stop as soon as the first closing `]` is found.

Reply:
@aojea: okay to lift the hold?

Reply:
I hate regex 😄

Reply:
> @aojea: okay to lift the hold?

yeah. added a comment, it seems my browser. didn't show me this comment before Xd